templates/init/storage/storage.rules

```diff
@@ -1,4 +1,8 @@
 rules_version = '2';
+
+// Craft rules based on data in your Firestore database
```

Comment:
Consider including a relevant docs links here?

Reply:
Ah good point. This copy was approved with PM so let me double check about the link and get back to this

Reply:
Please move this into a `match` block otherwise it won't work when you uncomment it as-is

Reply:
It's meant not to be uncommentable like that (to prevent accidentally opening things up)

Reply:
I'm going to go ahead and merge this as-is. These rules match the Console and in Console there's a banner already with a link to the docs.